requirements/runtime.txt

```diff
@@ -1,11 +1,13 @@
 #
-# This file is autogenerated by pip-compile with Python 3.12
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Pip-tools must always be executed under the same platform/runtime. Using a different one often results in transitive dependencies for different cases being removed or added.

Reply:
Fixed

Reply:
Thanks! Marking as resolved.